markets/hydration/index.ts

```diff
@@ -92,10 +94,25 @@ export const HydrationConfig: IAaveConfiguration = {
       WETH: "0xBd763043861CAF4E7e4E7Ffe951A03dF2Ea7E5AC",
       WBTC: "0xC9cCBe99bdD9538871f9756Ca5Ea64C2267cb0a7",
       DOT: "0x422E745797EC0Ef399c17cE3E2348394F2944727",
-      VDOT: "0x1B4A88Ce5A6c6878De2aC19694b2523e14E67eB6",
+      VDOT: "0x234F96059d628Da80B76A40c0E50a9D16a8F3191",
       //TBTC: "0x5d8320f3ced9575d8e25b6f437e610fc6a03bf52",
     },
   },
+  IncentivesConfig: {
+    [eHydrationNetwork.hydration] : {},
+    [eHydrationNetwork.nice] : {
+      DOT: [{
+        emissionPerSecond: BigNumber.from("413359788"),
+        duration: 1209600,
+        asset: "DOT",
```

Comment:
rename `asset -> reserve` to be clear this is identifying incentivized reserve pool of that asset